tests/indent/t32/if_block.cmm

```diff
@@ -0,0 +1,49 @@
+IF &a
+  STOP
+
+IF (TRUE())
+(
+  BREAK
+)
+
+IF (&b+CouNT())
+(
+  continue
+)
+
+IF FOUND()
+  STOP
+ELSE
+  CONTinue
+
+IF &c
+  CONTinue
+ELSE IF FALSE()
+  Break
+ELSE
+  stop
+
+IF &d
+(
+  STOP
+)
+ELSE IF &e
+; comment A
+(
+  CONTINUE
+)
+ELSE
+; comment B
+(
+  BREAK
+)
+
+IF &f
+  IF &g
+    stop
+  ELSE
+    IF &h
+  (
+    continue
+  )
```

Comment:
Tbh, if-else chaining has been a problem for god knows how long. Without block scope delimiters it's impossible to know whether this is the elseif of the if inner or outer.

That said, this is fine.

Reply:
Thank you for your feedback. Unfortunately, the TRACE32 scripting language has a lot of rough edges and that is surely one of them. I have added quite a few tests that I would consider as "nice-to-have", because I am still a bit unsure what the right expectation for the automatic indentation feature should be.